### ISSUE TITLE
 Class 'AMQPLibConnector' not found

### ISSUE THREAD

**Issue by SirbitoX** [gjedeer/celery-php#58]
Hi,

I'm trying to run the `test.php` file on Windows 7 OS/PHP 5.5.19 and I installed all dependencies using `composer.json` file. But when I run `php test.php` command I got:

> `Fatal error: Class 'AMQPLibConnector' not found in \path\to\celery-php\amqp.php on line 55`

And I realized that this error comes up with:

``` php
if(class_exists('PhpAmqpLib\Connection\AMQPConnection'))
{
    require_once('amqplibconnector.php');
    require_once('amqplibconnectorssl.php');

}
```

Where it couldn't find `PhpAmqpLib\Connection\AMQPConnection` class. I think something with namespacing is wrong. Do you have any idea how should I fix it?

p.s. I tried to fix it by removing the "if" condition but I got the similar error in another line of code.

**Comment by smuuf** [comment]
Second.

**Comment by gjedeer** [comment]
I'm fed up with composer bug reports. It seems like people are submitting Composer-related patches doing contradictory things. It's going back and forth, when somebody fixes a problem they have, it seems to always break dependencies for someone else.

Since I'm no authority when it comes to Composer, I'm not accepting any more of them. If someone who knows Composer very well is reading this, and willing to take responsibility of that part of Celery-PHP (handling bugs, preparing a test case, making sure it doesn't break non-composer installs etc), please contact me.

Previous Composer-related bugs:
* https://github.com/gjedeer/celery-php/pull/46
* https://github.com/gjedeer/celery-php/pull/40
* https://github.com/gjedeer/celery-php/issues/38
* https://github.com/gjedeer/celery-php/issues/34
* https://github.com/gjedeer/celery-php/issues/15

**Comment by gjedeer** [comment]
And in regard to your particular problem: just use it without composer until it's fixed, require_once celery.php and you're done.

**Comment by jdufresne** [comment]
This should now be fixed in commit 099ffdacda3919862e3094578554a4e833b0fd66. If you're willing, please install the latest dev version to test. Please let me know if you're still experiencing the same  issues.
